### DISCUSSION TITLE
ADMDatamart on model export data on local machine

### DISCUSSION THREAD

**Discussion by sushds** [pegasystems/pega-datascientist-tools#148] [Q&A]
Hi team, I am facing issues when I am trying to use the ADMDatamart function for the exports from my local machine. I am running out of memory and getting this error: MemoryError: Unable to allocate output buffer.

For reference I am just trying to get a datamart object as I need to look at some quick metrics. I have been able to export json file using readDSExport but as you know it provides raw data so it will take me quite some time to actually reach the analysis I am trying to do.

Hence, I tried using dm = ADMDatamart('my-path', , import_strategy='lazy') where my-path has string value of the directory where I have the zipped exports.

I was hoping to use dm t explore the pre-made functions of pdstools. Maybe you have encountered this issue before? Are there any pointers? Similar memory issue while trying to read the json file got resolved as readDSExport uses polars. Not sure what I could do here.

Please let me know if you would need more information from me, thanks!

(FYI, I work at bol.com)

**Comment by StijnKas** [comment]
Hi @sushantkhare, I've created an issue for this because I think it'll get more visibility: https://github.com/pegasystems/pega-datascientist-tools/issues/149
